models/order/src/order-from-cart-draft/presets/sample-data-fashion/jamie-cart-01.ts

```diff
@@ -0,0 +1,21 @@
+import { CartDraft } from '@commercetools-test-data/cart';
+import type { TCartDraft } from '@commercetools-test-data/cart';
+import { KeyReference } from '@commercetools-test-data/commons';
+import * as OrderFromCartDraft from '../../';
+import { constants } from '../../..';
+import { TOrderFromCartDraftBuilder } from '../../../types';
+
+const jamieCartDraft01 = CartDraft.presets.sampleDataFashion
+  .jamieDoe01()
+  .build<TCartDraft>();
+
+const jamieCart01 = (): TOrderFromCartDraftBuilder =>
+  OrderFromCartDraft.presets
+    .empty()
+    .version(null!)
```

Comment:
So this is interesting. This is the version of the cart, which will need to fetch and give to this preset. Perhaps for all of these, the builder can take version as a parameter? Would love others thoughts on this.

Reply:
Something like this?
```
const jamieCart01 = (versionNumber: number): TOrderFromCartDraftBuilder =>
  OrderFromCartDraft.presets
    .empty()
    .version(versionNumber)
```

Reply:
I like it - doesn't affect folks that don't need these presets, so we keep the pattern difference hidden.